android/pytorch-native/README.md

```diff
@@ -0,0 +1,137 @@
+
+# Pytorch Android Build Procedure
+## Linux (Ubuntu 20.04) android-sdk install
+```
+sudo apt-get install android-sdk
+export ANDROID_SDK_ROOT=/usr/lib/android-sdk
+sudo chown -R ubuntu:ubuntu $ANDROID_SDK_ROOT
+```
+
+## Mac android-sdk install
+```
+brew install android-sdk
+export ANDROID_SDK_ROOT=/Users/$USER/Library/Android/sdk
+sudo chown -R $USER $ANDROID_SDK_ROOT 
+```
+<br>
+<br>
+<br>
+<br>
+<br>
+<br>
+
+
+## Install Command line only tools
+Find latest command line only tools: [https://developer.android.com/studio#downloads](https://developer.android.com/studio#downloads:~:text=Command%20line%20tools%20only)
+
+```
+mkdir -p $ANDROID_SDK_ROOT/cmdline-tools
+cd $ANDROID_SDK_ROOT/cmdline-tools
+
+# From link above, set ANDROID_CLT equal to the name of Linux's latest SDK tools package release
+export ANDROID_CLT=commandlinetools-linux-8512546_latest.zip
+
+curl -O https://dl.google.com/android/repository/${ANDROID_CLT}
+unzip $ANDROID_CLT
+rm $ANDROID_CLT
+
+mv cmdline-tools tools
+```
+<br>
+
+## Install Android NDK
+See GitHub actions to ensure latest NDK_VERSION: [https://github.com/deepjavalibrary/djl/blob/master/.github/workflows/native_s3_pytorch_android.yml](https://github.com/deepjavalibrary/djl/blob/master/.github/workflows/native_s3_pytorch_android.yml)
+
+```
+export NDK_VERSION=21.0.6113669
+echo "y" | sudo ${ANDROID_SDK_ROOT}/cmdline-tools/tools/bin/sdkmanager --install "ndk;${NDK_VERSION}"
+```
+<br>
+
+## Build PyTorch Android native binary from source
+See: [https://github.com/deepjavalibrary/djl/blob/master/.github/workflows/native_s3_pytorch_android.yml](https://github.com/deepjavalibrary/djl/blob/master/.github/workflows/native_s3_pytorch_android.yml)
+
+Prerequisites:
+* If you do not have `cmake` installed, run `brew install cmake`
+* If you do not have the python library `pyyaml` installed, run `pip install pyyaml`
+* If you have `python3` installed, run `export PYTHON=python3`
```

Comment:
I would just include this into the script. Most systems nowadays should have python3. Maybe keep the comment to remove if you don't have python3